roles/ceph-ansible/README.rst

```diff
@@ -8,7 +8,11 @@ As it is written in ``tasks/main.yml`` ceph-ansible package should be
 preinstalled or available in enabled yum repository.
 
 Example usage of configured ceph-ansible:
-# cd to ceph-ansible dir so relative paths can be used
-$ cd ``{{ home_dir }}/ceph-ansible from defaults/main.yml``
-# /tmp/ansible.cfg is from {{ ansible_conf }} from defaults/main.yml
-$ ANSIBLE_CONFIG=/tmp/ansible.cfg ansible-playbook -i ~/ceph_inventory site.yml
+
+.. code:: bash
+
+  # cd to ceph-ansible dir so relative paths can be used
+  # default value of variable $base_dir is in defaults/main.yml
+  $ cd $base_dir/ceph-ansible from defaults/main.yml
```

Comment:
Thanks, now it make sense. Maybe just remove the ``from defaults/main.yml`` from the command.